apps/protailwind/.env.development

```diff
@@ -22,4 +23,6 @@ NEXT_PUBLIC_SANITY_API_VERSION="2022-07-29"
 CASTINGWORDS_TRANSCRIPT_ORDERING_ENABLED=true
 NEXTAUTH_URL=http://localhost:3017
 EMAIL_SERVER_HOST=smtp.postmarkapp.com
-EMAIL_SERVER_PORT=587
\ No newline at end of file
+EMAIL_SERVER_PORT=587
+NEXT_PUBLIC_DEFAULT_PRODUCT_ID=pt_60eb2f82-80ad-4bcf-8434-f34e81df7bbf
```

Comment:
these workshops are gonna be sold individually so this default concept is less relevant for now